### PR TITLE
TC-CNET-1.3 yaml

### PR DESCRIPTION
**Pull request by cecille**
#### Problem
Test wasn't automated

#### Change overview
automates test

#### Testing
chip-tool and all-clusters-app. Also covered by CI.

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml [hunk 1]
```diff
@@ -23,111 +23,114 @@ config:
...
-      verification: |
-          ./chip-tool networkcommissioning read cluster-revision  1 0
-
-          Verify on the TH Log:
-
-          [1649244866.275684][30440:30446] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0031 Attribute 0x0000_FFFD DataVersion: 2702518944
-          [1649244866.277721][30440:30446] CHIP:TOO:   ClusterRevision: 1
-          [1649244866.277827][30440:30446] CHIP:EM: Sending Standalone Ack for MessageCounter:10092551 on exchange 64234i
-      disabled: true
+      cluster: "Network Commissioning"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml:35]: Could just set that as the default value in the test config section above, right?

File: src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml [hunk 2]
```diff
@@ -23,111 +23,114 @@ config:
...
+          constraints:
+              type: list
+              contains:
+                  [
+                      0xFFFD,
+                      0xFFFC,
+                      0xFFFB,
+                      0xFFF9,
+                      0xFFF8,
+                      0x00,
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml:87]: As a followup, we could have a read of this conditioned on `F00 | F01` that checks for `ScanMaxTimeSeconds` and `ConnectMaxTimeSeconds`, right?

**Review by bzbarsky-apple — APPROVED** [review]
Thank you for doing these!

File: src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml (hunk 2 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml:87]: Yes, but that's not part of this test plan - 1.3 covers only the global attrs.

Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml:87]: oh, sorry, yeah, I misunderstood. Yes, I like that

**Review by arkq — APPROVED** [review]

**Comment by woody-apple** [comment]
@cecille note: Needs restyle

**Review by jmeg-sfy — APPROVED** [review]

**Comment by cecille** [comment]
We decided previously not to merge this PR as it was too close to the test deadline. Looks like it's a bit out of date now, but I'll get it updated.

**Comment by cecille** [comment]
already done.

**Pull request closed without merging**